app/Repositories/CommentLikeUserRepository.php

```diff
@@ -39,19 +39,21 @@ public function delete(CommentLikeUser $target): bool
     }
 
     //ログインユーザーがイイねをしているいかを判定するメソッド。
-    public function isLikedBy($comment,int $id): bool {
-        return CommentLikeUser::where('comment_id', $comment->id)->where('is_like', TRUE )->where('user_id', $id )->first() !==null;
+    public function isLikedBy(int $comment_id, int $id): bool
+    {
+        return CommentLikeUser::where('comment_id', $comment_id)->where('is_like', true)->where('user_id', $id)->first() !==null;
```

Comment:
```suggestion
        $commentLikeUser = $this->commentLikeUserRepo::where('comment_id', $comment_id)
            ->where('is_like', true)
            ->where('user_id', $id)
            ->first();
            
             return !is_null($commentLikeUser)
```

- 読みやすさ重視で改行を入れてほしい
- null は !== 比較しない
- メンバ変数の repo から実行してほしい (モデルのファサードは使わない)
- できれば where 条件は配列でまとめてほしい